app/helpers/commodities_helper.rb

```diff
@@ -216,17 +216,25 @@ def abbreviate_commodity_code(commodity)
   end
 
   def abbreviate_code(code)
-    case code.gsub(/0*\z/, '').length
+    only_code = code_without_subheading(code)
+
+    case only_code.gsub(/0*\z/, '').length
+    when 9..10
```

Comment:
I guess you mean this could be capture in the `else`,
yes, that's true ... I think I wrote it just to be more explicit,
even if the result is the same as the `else` branch, the case is a different one.